src/config.ts

```diff
@@ -125,6 +125,12 @@ export interface Config {
    */
   ignoreContextHeader?: boolean;
 
+  /**
+   * The ID of the Google Cloud Platform project with which traces should
+   * be associated.
+   */
+  projectId?: string;
```

Comment:
Qq. Shouldn't this be required? What happens if the project id is not given?

Reply:
It's not required. Project ID can also be either read from the metadata service, or the environment.

Reply:
On that note, let me add a comment to further elaborate on this.

Reply:
Thanks.